epam_task_02/task_02/my_text_reader/src/main/java/entity/TextComponent.java

```diff
@@ -0,0 +1,33 @@
+package main.java.entity;
+
+public interface TextComponent {
+    String printText();
+
+    TextType getTextType();
+
+    public enum TextType {
```

Comment:
I don't think that it is a good idea to place enum in Interface. But it is up to your opinion